docs/source/api/notebooks/index.rst

```diff
@@ -5,11 +5,15 @@ API Programming Examples
 
+.. note:: All runnable Jupyter notebooks and related python files can be downloaded from the `GraphStorm Github notebook repository <https://github.com/awslabs/graphstorm/tree/main/docs/source/api/notebooks>`_.
+
 These notebooks all use the same ACM data as discussed in the :ref:`User Your Own Data Tutorial<use-own-data>`. Users create the required dataset and explore details of it by running the `Notebook 0: Data Prepare <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_0_Data_Prepare.ipynb>`_ notebook.
 
-The `Notebook 1: NC Pipeline <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_1_NC_Pipeline.ipynb>`_ provides an example that demonstrates how to reproduce the node classification training and inference pipeline that is identical as the :ref:`Step 3: Launch training and inference scripts on your own graphs<launch_training_oyog>` does. While the `Notebook 2: LP Pipeline <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_2_LP_Pipeline.ipynb>`_ provides another example that use the same dataset, but conduct pipelines for link prediction training and inference. The `Notebook 3: Model Variants <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_3_Model_Variants.ipynb>`_ shows how to reproduce GraphStorm built-in model variants by using existing APIs.
+The `Notebook 1: Node classification Pipeline <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_1_NC_Pipeline.ipynb>`_ provides an example that demonstrates how to reproduce the node classification training and inference pipeline that is identical as the :ref:`Step 3: Launch training and inference scripts on your own graphs<launch_training_oyog>` does. While the `Notebook 2: Link Prediction Pipeline <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_2_LP_Pipeline.ipynb>`_ provides another example that use the same dataset, but conduct pipelines for link prediction training and inference. The `Notebook 3: Model Variants <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_3_Model_Variants.ipynb>`_ shows how to reproduce GraphStorm built-in model variants by using existing APIs. 
+
+To help users better understand how to modify GraphStorm for their specific requirements, the `Notebook 4: Customized Models <https://github.com/awslabs/graphstorm/blob/main/docs/source/api/notebooks/Notebook_4_Customized_Models.ipynb>`_ provides an example of a customized `RGAT` encoder and how to embed it into the GraphStorm model architecture.
```

Comment:
It is better to keep the notebook names consistent.

Reply:
Yes, aligned all notebook with their titles.